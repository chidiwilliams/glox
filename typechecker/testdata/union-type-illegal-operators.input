var x: string | number = "hello";
var y: string | number = 32;

print x / y;
