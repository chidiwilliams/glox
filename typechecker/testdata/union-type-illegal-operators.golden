type error on line 4: unexpected type: string | number, allowed: number
